test/checkout.int.t.sol

```diff
@@ -1039,13 +946,157 @@ contract Checkout_swapWithoutFeeSpecialOrder is Checkout {
 
     assertEq(
       _bpNori.balanceOf(_namedAccounts.supplier),
-      (certificateAmount * customPriceMultiple) / 100 / 2 // divide to account for price multiple scale and then holdback percentage of 50%
+      (certificateAmount * customPriceMultiple) / 100 / 2 // divide to account for price multiple scale and holdback percentage of 50%
     );
     assertEq(_bpNori.balanceOf(_namedAccounts.feeWallet), 0);
   }
 }
 
-contract Checkout_swapFromSupplierWithoutFeeSpecialOrder is Checkout {
+contract Checkout_swapWithoutFeeSpecialOrder_specificVintages is Checkout {
+  uint256 ownerPrivateKey = 0xA11CE;
+  address owner = vm.addr(ownerPrivateKey);
+  uint256 customFee = 5;
+  uint256 certificateAmount = 2.5 ether;
+  uint256[] vintages = [2019, 2020];
+  uint256 _priceMultiple = 2000;
+
+  function setUp() external {
+    /**
+     * Supplier 1: 2018 vintage
+     * Supplier 2: 2018, 2019 and 2020 vintage
+     * Supplier 3: 2019 vintage
+     * All removals are 1 tonne
+     * Certificate amount is 2.5 tonnes
+     * Fulfillment should result in all of supplier2's removals being used, and .5 tonne of supplier3's removals
+     */
+    _removalIds.push(
```

Comment:
Looking at how this new seed function is being used, it seems like this should have could have been more consistent if it extended this pattern, but also points out how there are too many ways to seed removals creating test setup to vary between tests. 🙈

https://github.com/nori-dot-eco/contracts/blob/a96d8d6413907b674a119cc1d53afe589831470f/test/checkout.int.t.sol#L195